docs/Generators/Ibex.rst

```diff
@@ -0,0 +1,14 @@
+Ibex Core
+====================================
+
+`Ibex <https://github.com/lowRISC/ibex>`__ is a parameterizable RV32 embedded core written in SystemVerilog, currently maintained by lowRISC.
+The `Ibex core` is wrapped in an `Ibex tile` so it can be used with the `Rocket Chip SoC generator`.
+The core exposes a custom memory interface, interrupt ports, and other misc. ports that are connected from within the tile to TileLink buses and other parameterization signals.
+
+.. Warning:: The Ibex mtvec register is 256 byte aligned. When writing/running tests, ensure that the trap vector is also 256 byte aligned.
+
+.. Warning:: The Ibex reset vector is located at 0x80.

```

Comment:
Isn't the more appropriate thing to say is that the reset vector is located at ADDRESS+OFFSET where OFFSET is 0x80.

Reply:
Yeah, that sounds more correct. I'll fix it.